builder/builder-next/builder.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/containerd/containerd/remotes/docker"
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/backend"
+	apitime "github.com/docker/docker/api/types/time"
```

Comment:
can you use `timetypes` for this one? Looks like that's currently the alias we use everywhere.

Reply:
Ah yes, I remember I wanted to check how this package is aliased in other places but forgot 🙈 
Fixed, thanks!